dataactbroker/handlers/accountHandler.py

```diff
@@ -387,6 +387,10 @@ def updateUser(self, system_email):
         if requestDict.exists("is_active"):
             is_active = bool(requestDict.getValue("is_active"))
             if not self.isUserActive(user) and is_active:
+                # Reset password count to 0
+                self.resetPasswordCount(user)
+                # Reset last login date so the account isn't expired
+                self.interfaces.userDb.updateLastLogin(user, unlock_user=True)
```

Comment:
Just FYI, the main blockers to MAX right now are the MAX team bothering to
respond to me in an email, and getting .gov emails for the dev team
(required for MAX log in).

On Fri, May 27, 2016 at 2:28 PM, Becky Sweger <notifications@github.com>
wrote:

> In dataactbroker/handlers/accountHandler.py
> <https://github.com/fedspendingtransparency/data-act-broker-backend/pull/63#discussion_r64948162>
> :
>
> > @@ -387,6 +387,10 @@ def updateUser(self, system_email):
> >          if requestDict.exists("is_active"):
> >              is_active = bool(requestDict.getValue("is_active"))
> >              if not self.isUserActive(user) and is_active:
> > +                # Reset password count to 0
> > +                self.resetPasswordCount(user)
> > +                # Reset last login date so the account isn't expired
> > +                self.interfaces.userDb.updateLastLogin(user, unlock_user=True)
>
> Ok, let's put it as a technical debt task and use your method of the
> unlock_user parameter as a stop-gap.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fedspendingtransparency/data-act-broker-backend/pull/63/files/e81d5afe61c941c6cf59b6eaceefd00e48eae014#r64948162>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEnIzZB7v9qezEu-eMJoztlvwpiB0Mzks5qFze_gaJpZM4IotEa>
> .
>



-- 
Kaitlin Devine
Treasury/18F

Need to schedule a meeting? Check out my Calendly page
<https://calendly.com/kaitlin-devine>